ubuntu1704-desktop.json

```diff
@@ -0,0 +1,15 @@
+{
+  "_comment": "Build with `packer build -var-file=ubuntu1704-desktop.json ubuntu.json`",
+  "vm_name": "ubuntu1704-desktop",
+  "desktop": "true",
+  "locale": "en_US.UTF-8",
+  "cpus": "1",
+  "disk_size": "130048",
```

Comment:
This attribute confused me. I'm not sure if it's the size of the iso or the size you want the VM?